src/pyFAI/io/integration_config.py

```diff
@@ -443,23 +447,22 @@ def pop_method(self, default=None):
 @mydataclass
 class WorkerConfig:
     """Class with the configuration from the worker."""
-    application: str="worker"
+    application: str = "worker"
     version: int = CURRENT_VERSION
-    poni: object = None
+    poni: PoniFile = None
```

Comment:
:+1: 

No need to put `| None` (needs `from __future__ import annotations`)?

```suggestion
    poni: PoniFile | None = None
```

Reply:
Since I am not checking types :)